ambry-rest/src/test/java/com/github/ambry/rest/NettyResponseChannelTest.java

```diff
@@ -658,7 +659,8 @@ public void channelInactiveTest() {
     EmbeddedChannel channel = new EmbeddedChannel(chunkedWriteHandler);
     NettyResponseChannel nettyResponseChannel =
         new NettyResponseChannel(new MockChannelHandlerContext(channel), new NettyMetrics(new MetricRegistry()),
-            new PerformanceConfig(new VerifiableProperties(new Properties())));
+            new PerformanceConfig(new VerifiableProperties(new Properties())),
+            new NettyConfig(new VerifiableProperties(new Properties())));
```

Comment:
Minor: please create VerifiableProperties once since it's being used multiple times.

Reply:
Done.